app/javascript/components/schedule-form/schedule-form.schema.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable camelcase */
+import { componentTypes } from '@@ddf';
+import {
+  nameField, descriptionField, activeField, actionField, attributeValueFields, runField, filterField, targetField, zoneField, plainField, systemField,
+  objectMessageField, objectRequestField, objectTypeField, objectItemField, timerValueField, timezoneField, startDateField, startTimeField,
+} from './schedule-form-fields';
+
+function createSchema(_recordId, actionOptions, filterOptions, data, setData) {
```

Comment:
this can be like -

`export const createSchema = (_recordId, actionOptions, filterOptions, data, setData) => {`


and we can remove the last export statement